common/src/main/java/com/linecorp/centraldogma/common/QueryType.java

```diff
@@ -46,4 +52,22 @@
     public Set<EntryType> supportedEntryTypes() {
         return supportedEntryTypes;
     }
+
+    /**
+     * Returns a {@link QueryType} from the specified {@code value}. If none of query type is matched,
+     * this will return {@link #IDENTITY}.
+     */
+    public static QueryType parse(@Nullable String value) {
+        if (isNullOrEmpty(value)) {
+            return IDENTITY;
+        }
+
+        final String queryType = Ascii.toUpperCase(value);
```

Comment:
Seems like uppercasing here is unnecessary because you used `equalsIgnoreCase`?

Reply:
Oops~!!